### PR TITLE
view operation and phases 2.0

### LINKED ISSUE #1: Single Operation view
List Jobs and runs
View location on Map

### PR DESCRIPTION
**Pull request by soofstad**
- Added .env-template
- Added the "ForecastDS" as a visible DS in DMT Application
- Updated FoR Blueprints
- Move main pages to "Pages/" from "Components/"
- Detailed OperationView with Comments and Phase View
- Removed all things weather related

closes #1

### CODE REVIEW

File: web/custom-plugins/forecast-of-response/src/components/Operations/OperationDetails.tsx [hunk 1]
```diff
@@ -1,106 +1,132 @@
...
+                <Table.Cell>
+                  {new Date(phase.start).toLocaleString(local)}
+                </Table.Cell>
+                <Table.Cell>
+                  {new Date(phase.end).toLocaleString(local)}
+                </Table.Cell>
+                <Table.Cell>
+                  <FlexWrapper>
+                    {phase.status}
+                    <StatusDot status={OperationStatus.ONGOING} />
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/Operations/OperationDetails.tsx:112]: can we use phase.status for the StatusDot component?

File: web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          }}
+        >
+          <Button
+            variant="outlined"
+            color="danger"
+            onClick={() => setVariables(defaultVars)}
+          >
+            Reset
+          </Button>
+          <Button>Create</Button>
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx:102]: maybe add todo comment or tooltip for button if functionality is not implemented yet

File: web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            )}
+            {simulationConfig.simulations.map(
+              (simulation: TSimulation, index: number) => (
+                <SimulationRow
+                  key={index}
+                  even={index % 2 === 0}
+                  selected={index === selectedSim}
+                  onClick={() => setSelectedSim(index)}
+                >
+                  <div>{new Date(simulation.started).toLocaleString()}</div>
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx:157]: maybe we can use the 'en-GB' locale here

File: web/custom-plugins/forecast-of-response/src/components/Comments.tsx [hunk 4]
```diff
@@ -49,7 +71,81 @@ export const CompactCommentView = ({ comment }: ICompactCommentView) => {
...
+  const { comment, even } = props
+  const uniqueAuthorColor = colorFromString(comment.author)
+  return (
+    <CommentWrapper color={uniqueAuthorColor} even={even}>
+      <CommentHeaderWrapper>
+        <IconWrapper color={uniqueAuthorColor}>&#9679;</IconWrapper>
+        <b>{comment.author}</b>
+      </CommentHeaderWrapper>
+      <b style={{ width: 'fit-content' }}>
+        {new Date(comment.date).toLocaleString()}
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/Comments.tsx:89]: we can probably use the locale 'en-GB' here

File: web/custom-plugins/forecast-of-response/src/components/Comments.tsx [hunk 5]
```diff
@@ -49,7 +71,81 @@ export const CompactCommentView = ({ comment }: ICompactCommentView) => {
...
+export const CommentInput = (props: { operationId: string }) => {
+  const { operationId } = props
+  const [message, setMessage] = useState<string>('')
+  const { token, userData } = useContext(AuthContext)
+  const dmssAPI = new DmssAPI(token)
+
+  function handlePost() {
+    // TODO: When we can import model contained data, remove name from Comment
+    const commentName = crypto.randomUUID()
+    dmssAPI.generatedDmssApi.explorerAddToParent({
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/Comments.tsx:113]: When i tested locally i got a 400 Bad request error from this function: "BadRequestException: Attribute not specified on parent".

**Review by KristianKjerstad — CHANGES_REQUESTED** [review]
good! just a few minor things...

File: web/custom-plugins/forecast-of-response/src/components/Operations/OperationDetails.tsx (hunk 1 above)
Comment (soofstad) [reply web/custom-plugins/forecast-of-response/src/components/Operations/OperationDetails.tsx:112]: fixed!

File: web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx (hunk 3 above)
Comment (soofstad) [reply web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx:157]: fixed. Used "navigator.local"

File: web/custom-plugins/forecast-of-response/src/components/Comments.tsx (hunk 4 above)
Comment (soofstad) [reply web/custom-plugins/forecast-of-response/src/components/Comments.tsx:89]: fixed!

**Pull request merged**